package.json

```
@@ -44,10 +47,11 @@
     "xo": "^0.37.0"
   },
   "scripts": {
-    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
+    "format": "remark . -qfo --ignore-pattern test/ && prettier . -w --loglevel warn && xo --fix --ignore types",
```

the test folder wasn't modified, it shouldn't need to be ignored
```suggestion
    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix --ignore types",
```